vmdb/lib/workers/worker_base.rb

```diff
@@ -88,16 +90,14 @@ def self.wait_for_worker_monitor=(val)
 
   def my_monitor_started?
     return @monitor_started unless @monitor_started.nil?
-    server = MiqServer.my_server(true)
     return false if     server.nil?
-    return false unless server.started?
+    return false unless server.reload.started?
     @monitor_started = true
   end
 
   def worker_monitor_drb
-    server = MiqServer.my_server(true)
     raise "#{self.log_prefix} No MiqServer found to establishing DRb Connection to" if server.nil?
-    drb_uri = server.drb_uri
+    drb_uri = server.reload.drb_uri
```

Comment:
I think you're right, reload shouldn't be necessary since you can't get to this method until the server has started (via above method `my_monitor_started?` and the drb_uri is set before the server is marked as started, see [here](https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/miq_server.rb#L187-L191)